serokell-style.md

```diff
@@ -485,6 +530,8 @@ foo = do
         return ()
 ```
 
+Use `-XMultiwayIf` only if you need complex `if-then-else` inside `do`-blocks.
```

Comment:
Imo, multiway `if` looks better than regular `if` so I use never use regular `if` 🤷‍♂️

Reply:
I agree with @ChShersh on this rule.